### PR TITLE
Show active filters at top on a small screen

### PR DESCRIPTION
**Pull request by leb2dg**
Resolves: #67 

screen shot 2016-08-10 at 1 34 44 pm

### CODE REVIEW

File: app/controllers/discover.js [hunk 1]
```diff
@@ -240,6 +240,16 @@ export default ApplicationController.extend({
             this.set(type, newValue);
         },
 
+        removeFilter(type, filterValue) {
+            let currentValue = getSplitParams(this.get(type));
+            currentValue = currentValue ? currentValue : [];
```
Comment (aaxelb) [inline app/controllers/discover.js]: These two lines could be condensed to `let currentValue = getSplitParams(this.get(type)) || [];`, which would be nicer.

File: app/templates/discover.hbs [hunk 2]
```diff
@@ -26,7 +26,7 @@
         </div>
     </div>
     {{#if queryBody}}
-        <div class="row" style="text-align: center; font-size:75%; margin-top:15px">
+        <div class="row hidden-xs" style="text-align: center; font-size:75%; margin-top:15px">
```
Comment (aaxelb) [inline app/templates/discover.hbs]: It looks like the filter sidebar is completely hidden on small screens. Is it possible to add filters on mobile?

File: app/templates/discover.hbs [hunk 3]
```diff
@@ -70,6 +70,19 @@
             </select>
         </div>
     </div>
+    <div class="row visible-xs">
```
Comment (aaxelb) [inline app/templates/discover.hbs]: Do you think listing the filters at the top could be useful on larger screens, too? Or is that too redundant?

File: app/templates/discover.hbs (hunk 2 above)
Comment (leb2dg) [inline app/templates/discover.hbs]: It is by clicking on the results.

File: app/templates/discover.hbs (hunk 3 above)
Comment (leb2dg) [inline app/templates/discover.hbs]: Maybe? We could easily show them.

File: app/templates/discover.hbs (hunk 2 above)
Comment (leb2dg) [inline app/templates/discover.hbs]: Also, be transitioning from the homepage or sources page.

Comment (aaxelb) [inline app/templates/discover.hbs]: I feel like we don't want to hide features from mobile users, though (now that we're acknowledging they exist...). Could the sidebar be expandable?

I guess they can filter by anything in the search bar if they know the syntax, though, so the feature is still technically available, if opaque.

Comment (leb2dg) [inline app/templates/discover.hbs]: I can't think of a way the user could see both the filter facets and the results at the same time -- the screen just isn't large enough. Although I agree that features shouldn't be hidden.

Comment (aaxelb) [inline app/templates/discover.hbs]: They couldn't see both, but what if (on `xs` screens) the facets were hidden by default, with a "Filter" button/link that toggles them? When displayed, they'd be stacked above the search results, which could be annoying to users but they'd have to ask for it.

Comment (aaxelb) [inline app/templates/discover.hbs]: Or maybe an "add filters..." link next to the active filters list

**Pull request merged**